custom_components/owlintuition/owl_intuition.yaml

```diff
@@ -1,12 +1,17 @@
 ####################################################
-# Owl Intution - Grid Energy Sensor                #
+# Owl Intuition - Energy Sensor                     #
 ####################################################
+
 template:
+  # State ensures:
+  #  - value is not set until both solar and electricity readings have been received and are no longer unknown
+  #  - reads the values in simple variables for elec and solar, to make if statements easier to read
+  #  - protects value going negative if one sensor resets to zero before the other at around midnight
   - trigger:
     - platform: state
       entity_id: sensor.owl_intuition_electricity_power
     sensor:
-      - name: "Owl Grid Power Now"
+      - name: "Owl Grid Energy Now"
```

Comment:
It's still power right

Reply:
It is actually! This template is not part of the "core" component, yet let's revert that